docs/Design/Rest Parameters.md

```diff
@@ -0,0 +1,143 @@
+# Rest Parameters
+
+A parameter that creates an array for all unassigned arguments.
+
+
+## Commands
+
+### `rest parameters`
+
+`rest parameters : type name`

```

Comment:
`variable` is *name, type*, value. `member variable declare` is privacy, *name, type*. Why switch the order here?